i18n/en.json

```
@@ -1835,10 +1837,12 @@
   "admin.reporting.workspace_optimization.cta.startTrial": "Start Trial",
   "admin.reporting.workspace_optimization.cta.upgradeLicense": "Upgrade to Enterprise",
   "admin.reporting.workspace_optimization.data_privacy.description": "Get better insight and control over your data.",
+  "admin.reporting.workspace_optimization.data_privacy.descriptionOk": "You seem to be using data retention and compliance features!",
   "admin.reporting.workspace_optimization.data_privacy.retention.description": "Organizations in highly regulated industries require more control and insight with their data. We recommend starting an Enterprise trial to take advantage of Mattermost Data Retention and Compliance features.",
   "admin.reporting.workspace_optimization.data_privacy.retention.title": "Become more data aware",
   "admin.reporting.workspace_optimization.data_privacy.title": "Data Privacy",
   "admin.reporting.workspace_optimization.ease_of_management.description": "Make it easier to manage your Mattermost workspace.",
+  "admin.reporting.workspace_optimization.ease_of_management.descriptionOk": "For your current usage scale, manually managing users and guests seems apropriate!",
```

this differs from the defaultMessage in code. is that intentional?

```suggestion
  "admin.reporting.workspace_optimization.ease_of_management.descriptionOk": "For your current usage scale, manually managing users and guests seems appropriate!",
```